spec/features/admin/admin_sees_order_dashboard_spec.rb

```diff
@@ -82,7 +82,7 @@
     expect(page).to have_button "Mark as completed"
   end
 
-  scenario 'can filter order dashboard' do
+  xscenario 'can filter order dashboard' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.